ProxyStubGenerator/StubGenerator.py

```diff
@@ -785,9 +785,9 @@ def _FindLength(length_name, variable_name):
                         log.InfoLine(self.identifier, "'%s': @maxlength not specified for this inbound buffer, assuming same as @length" % self.trace_proto)
 
                 if (self.is_buffer and self.is_output and not self.max_length):
-                    raise TypenameError(self.identifier, "'%s': can't deduce maximum length of this outbound buffer, use @maxlength" % self.trace_proto)
+                    raise TypenameError(self.identifier, "'%s': can't deduce maximum length of this inbound buffer, use @maxlength" % self.trace_proto)
```

Comment:
Perhaps I'm just confused, but is this correct? self.is_output seems to be about output, but perhaps that is not the buffer from the message?

Reply:
Yes, precisely.